useCase/useCase.go

```diff
@@ -0,0 +1,40 @@
+package useCase
+
+import (
+	"strconv"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/app"
+	"github.com/Diegoplas/go-bootcamp-deliverable/utils"
+)
+
+func GetPokemonFromCSV(wantedIndex int) utils.PokemonData {
+
+	csvLines := app.CSVReader()
```

Comment:
As I mentioned on the csvReader function file, you would probably receive the Pokemons already parsed here, and an error, so remember to handle it